src/omotes_simulator_core/adapter/transforms/mappers.py

```diff
@@ -167,33 +149,17 @@ def to_entity(self, network: Network) -> Tuple[List[AssetAbstract], List[Junctio
                         index = [py_asset_temp.asset_id for py_asset_temp in py_assets_list].index(
```

Comment:
Kun je wat extra comments toevoegen? Het is soms lastig om te controleren wat de oude code aan het doen is.